src/dune_rules/compilation_context.ml

```diff
@@ -18,7 +19,7 @@ module Includes = struct
     let cmx_includes =
       Command.Args.memo
         (Resolve.Memo.args
-           (let+ libs = requires in
+           (let+ libs = requires in (* requires here represents the list of deps of the target library; it is equal to requires_compile 1.1*) 
```

Comment:
It's OK to comment the code to help yourself, but let's remember to remove the comment eventually.